global-api/main.py

```diff
@@ -95,6 +96,16 @@ def read_root():
     tags=["EDGAR"],
 )
 
+app.include_router(
+    catalogue_last_update_endpoint,
+    tags=["Datasource Catalogue"],
+)
+
+app.include_router(
+    catalouge_route,
+    tags=["Datasource Catalogue"],
+)
```

Comment:
I don't think you need to define a new router for each route necessarily, it can be one for each set of routes usually. But it's fine otherwise.